ONSdigital/dis-wagtail-alpha
# Hard-coded header and navigation

### What is the context of this PR?

Hard-coded header for the November prototype.
Duplicates the header macro from the design system and replaces `serviceLinks` with `navLinks`
Hard-codes the configuration for the navLinks in `base.html`

### How to review
- Check out the `feature/november-prototype-header` branch
- Open any page in a local build, and check that the header displays as per the designs at desktop and mobile
- Ensure that there is only one link in the header navigation - for now, this has a dummy value. We need to hard-code a link to a bulletin page at some point.
